.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ env:
   # An envar that signals to tests we are executing in the CI environment
   CONTINUOUS_INTEGRATION: true
   # allow overriding Maven command
-  MAVEN: ./mvnw
+  MAVEN: ./mvnw --offline
```

Comment:
i'd rather see this parent of MAVEN_FAST_INSTALL or something

if "all" maven invocations are `--offline` who would ever download stuff?

Reply:
I'm using `--offline` only as an experiment to see if we actually downloaded all dependencies.